src/plugins/dam/components/uploader/Uploader.jsx

```diff
@@ -252,6 +265,57 @@ class Uploader extends React.Component {
   }
 }
 
+class SaveButton extends React.Component {
```

Comment:
@digitalartcrew why does this need to be a separate component?

Reply:
This was so the button could have it's own state that wouldn't re-render the entire component

Reply:
https://recordit.co/b4lpOFGKts